alloydb/notebooks/batch_embeddings_update.ipynb

```diff
@@ -0,0 +1,1098 @@
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 22,
+      "metadata": {
+        "id": "fYKVQzv2cjcm"
+      },
+      "outputs": [],
+      "source": [
+        "import asyncpg\n",
+        "\n",
+        "import sqlalchemy\n",
+        "from sqlalchemy.ext.asyncio import AsyncEngine, create_async_engine\n",
+        "\n",
+        "from google.cloud.alloydb.connector import AsyncConnector, IPTypes\n",
+        "\n",
+        "async def init_connection_pool(connector: AsyncConnector, db_name: str, pool_size: int = 5) -> AsyncEngine:\n",
+        "    # initialize Connector object for connections to AlloyDB\n",
+        "    connection_string = f\"projects/{project_id}/locations/{region}/clusters/{cluster_name}/instances/{instance_name}\"\n",
+        "\n",
+        "    async def getconn() -> asyncpg.Connection:\n",
+        "        conn: asyncpg.Connection = await connector.connect(\n",
+        "            connection_string,\n",
+        "            \"asyncpg\",\n",
+        "            user=\"postgres\",\n",
+        "            password=password,\n",
+        "            db=db_name,\n",
+        "            ip_type=IPTypes.PUBLIC,\n",
+        "        )\n",
+        "        return conn\n",
+        "\n",
+        "    pool = create_async_engine(\n",
+        "        \"postgresql+asyncpg://\",\n",
+        "        async_creator=getconn,\n",
+        "        pool_size=pool_size,\n",
+        "        max_overflow=0,\n",
+        "    )\n",
+        "    return pool"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 23,
+      "metadata": {},
+      "outputs": [
+        {
+          "name": "stderr",
+          "output_type": "stream",
+          "text": [
+            "I0000 00:00:1729792450.162148 11765976 fork_posix.cc:77] Other threads are currently calling into gRPC, skipping fork() handlers\n"
+          ]
+        }
+      ],
+      "source": [
+        "from google.cloud.alloydb.connector import AsyncConnector\n",
+        "\n",
+        "connector = AsyncConnector()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "i_yNN1MnJpTR"
+      },
+      "source": [
+        "### Create a Database\n",
+        "\n",
+        "Nex, you will create database to store the data using the connection pool. Enabling public IP takes a few minutes, you may get an error that there is no public IP address. Please wait and retry this step if you hit an error!"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "7PX05ndo_AMc",
+        "outputId": "0931754a-aeb8-4895-e0b5-eeb01ffe5506"
+      },
+      "outputs": [],
+      "source": [
+        "from sqlalchemy import text, exc\n",
+        "\n",
+        "async def create_db(database_name, connector):    \n",
```

Comment:
This might be useful for a user if they want to re-use the same function, potentially to create multiple databases to test things out. 

I see more cons than pros of removing this function. What do you think?